### PR TITLE
Assignment added

### PR DESCRIPTION
**Pull request by timsinapranjal**

### CODE REVIEW

**Comment by prertik** [comment]
https://github.com/Nischalkhadka/Quality-Assurance/pull/7#issuecomment-573582889

File: QA Interns 2020/Assignments/Pranjal/API Testing.postman_environment.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"id": "5d8ea715-42e7-441b-8d48-cdfe6bb35ba0",
+	"name": "API Testing",
+	"values": [],
+	"_postman_variable_scope": "environment",
+	"_postman_exported_at": "2020-01-16T08:29:01.359Z",
+	"_postman_exported_using": "Postman/7.14.0"
+}
```
Comment (Nischalkhadka) [inline QA Interns 2020/Assignments/Pranjal/API Testing.postman_environment.json]: Add a new line

**Comment by Nischalkhadka** [comment]
Do resolve the conflicts here.

**Comment by Nischalkhadka** [comment]
Do review the files. I cannot merge this due to conflict.

**Pull request merged**